#144 fixed multinavigation comp dark themes

smcgrath0 opened the pull request:
**What does this implement/fix? Explain your changes.**

#144; Adjusted multinavigation dark theme colors, that won't affect other components.

**Does this close any currently open issues?**

#144 

**Where has this been tested?**
Local. 
Now v
<img width="375" alt="Screen Shot 2020-08-11 at 11 53 07 AM" src="https://user-images.githubusercontent.com/49845255/89936932-3ad26d80-dbc9-11ea-8562-34dd3d0858ae.png">

*Browser (version):* Chrome v84

smcgrath0 committed:
#144 fixed multinavigation comp dark themes

qial reviewed (changes requested):
Thanks! This works. However, the template.vue files are auto-generated through the hatch.js.
Here is the relevant line that creates that v-bind:class attribute:

https://github.com/headwirecom/themeclean-flex/blob/issues/144/fragments/navigation/hatch.js#L25

Could you remove this line from the hatch.js file and then do a `percli hatch navigation` to regenerate? That should be the only thing that needs to be changed to reflect your changes to the template.vue file.

smcgrath0 committed:
#144 added to hatch file

smcgrath0 commented:
I removed that line and hatched, but the previous work on this component wasn't hatched. Which is why I added another line in the hatch.js file. Also, all the attributes are the same, but the order changed.

reusr1 commented:
@smcgrath0 please update to the latest percli version - it seems you are hatching with an older version of the tool. I did a quick test of this and had some issues as the colors were still off (check all current palettes and default/dark/light styles.

reusr1 commented:
it also looks the navigation has issues when it is open and you start scrolling the page

reusr1 reviewed (changes requested):
see comments before

smcgrath0 committed:
#144 updated with latest version of percli hatch

smcgrath0 commented:
Fixed dark themes with latest version of percli hatch, can be merged.
